ext_mongodb.php

```diff
@@ -19,6 +19,110 @@ function bsonUnserialize(array $data) : void;
 {
 }
 
+/* == MONITORING ============================================================= */
+namespace MongoDB\Driver\Monitoring;
+
+/* ------ Building blocks for value obejcts */
+abstract class _CommandEvent
+{
+	private string                 $commandName;
+	private \MongoDB\Driver\Server $server;
+	private integer                $operationId;
+	private integer                $requestId;
+
+	private function __construct()
+	{
+		throw new \MongoDB\Driver\Exception\RunTimeException("Accessing private constructor");
```

Comment:
Is this just here to catch someone invoking the constructor using Reflection trickery?